ci/acm_subscription.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: acm-operator-group
+  namespace: advanced-cluster-management
+spec:
+  targetNamespaces:
+  - advanced-cluster-management
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: acm-operator-subscription
+  namespace: advanced-cluster-management
+spec:
+  channel: release-2.7
+  installPlanApproval: Automatic
+  name: advanced-cluster-management
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: advanced-cluster-management.v2.7.3
```

Comment:
Is this field needed?

Reply:
I am not sure, didn't try without it yet.
Assuming this is a dependency, don't we want to pin point its version instead of using the latest CSV?

Reply:
I think we could work with the latest 2.7 release. I would even consider using the `stable` channel if it exists.